### ISSUE TITLE
PHP notice is displayed related to bp_is_blog_page() fun and navbar is not visible 

### ISSUE THREAD

**Issue by pavanpatil1** [rtCamp/rtMedia#1853]
```
Enviornment: 
WordPress: v5.9.1
BuddyPress: v10.1.0
rtMedia: v4.6.10
BP rewrites: v1.0.0-beta2
```

PHP notice is displayed related to elated to bp_is_blog_page() fun after activating the BP rewrite plugin. 

`bp_is_blog_page() plugins\buddypress-media\app\main\controllers\template\RTMediaNav.php:88`

**Steps to reproduce:** 
Install and activate the [BP rewrites](https://bpdevel.wordpress.com/2022/03/12/bp-rewrites-1-0-0-beta2/) plugin and observed the issue. 

https://user-images.githubusercontent.com/44057535/158188465-7d4ce5a5-3aaf-4653-a9bf-d59794c64e38.mp4

**Comment by vishalkakadiya** [comment]
@thebengalboy This issue will be fixed with the `feature/bprewrite` branch, but we can't directly merge this to `pre-release` as some other things are still not working in this, once we fixed all then we can merge this to `pre-release`. Thanks!
CC @Pradeep1308

**Comment by pavanpatil1** [comment]
Closing this issue as fixed in #1887
